libbeat/scripts/generate_fields_docs.py

```diff
@@ -6,6 +6,8 @@
 
 
 def document_fields(output, section, sections, path):
+    if  "skipdocs" in section:
```

Comment:
double space between `if` and `"skipdocs"` makes CI style checker fail.

Reply:
Good catch! I think I am too accustomed to ignoring CI failures. :-P